src/main/java/racingcar/view/ResultView.java

```diff
@@ -17,13 +17,12 @@ public ResultView(PrintWriter output) {
         this.output = output;
     }
 
-    public void show(List<List<RecordDto>> records) {
+    public void showResultHeader() {
         output.println("실행 결과");
-        records.forEach(this::showAtStep);
     }
 
-    private void showAtStep(List<RecordDto> recordAtStep) {
-        recordAtStep.forEach(recordDto -> output.println(format("%s : %s", recordDto.getName(), String.join("", Collections.nCopies(recordDto.getRecord(), SYMBOL)))));
+    public void showAtStep(List<RecordDto> recordAtStep) {
+        recordAtStep.forEach(recordDto -> output.println(format("%s : %s", recordDto.getName(), String.join("", Collections.nCopies(recordDto.getPosition(), SYMBOL)))));
```

Comment:
```suggestion
      recordAtStep.forEach(recordDto ->
          output.println(
              format(
                  "%s : %s",
                  recordDto.getName(),
                  String.join("", Collections.nCopies(recordDto.getPosition(), SYMBOL))
              )
          )
      );
```
조금만 가독성을 신경써주시면 좋을 것 같아요!

Reply:
혹은 다음과 같이 표현할 수 있습니다.

```java
        recordAtStep
          .stream()
          .map(recordDto -> format(
            "%s : %s",
            recordDto.getName(),
            String.join("", Collections.nCopies(recordDto.getPosition(), SYMBOL))
          ))
          .forEach(output::println);
```

그리고 문자를 반복하여 이어 붙이는 기능은 다른 메소드로 만들면 어떨까요?
```java
        recordAtStep
          .stream()
          .map(recordDto -> format(
            "%s : %s",
            recordDto.getName(),
            repeat(SYMBOL, recordDto.getPosition())
          ))
          .forEach(output::println);
```
조금 더 직관적으로 이해할 수 있도록 만들면 좋을 것 같습니다.
아무래도 view의 경우 출력에 필요한 작업을 하다보니, 조금 난잡해질 수 있고 생각보다 변화가 자주 발생하는 구간이라서 최대한 알아보기 좋게 만드는게 중요합니다.